static/app/views/settings/project/projectReleaseTracking.tsx

```diff
@@ -121,18 +123,19 @@ class ProjectReleaseTracking extends AsyncView<Props, State> {
     return (
       <div>
         <SettingsPageHeader title={t('Release Tracking')} />
+        <TextBlock>
+          {t(
+            'Configure release tracking for this project to automatically record new releases of your application.'
+          )}
+        </TextBlock>
+
```

Comment:
this text block is now above the alert?
<img width="1081" alt="Screenshot 2023-04-28 at 14 34 14" src="https://user-images.githubusercontent.com/70817427/235258063-99f12a25-f37b-4c85-adfe-448bbc400511.png">

Reply:
I'll make it consistent at for my final run-through.